uktrade/data-hub-frontend
# feature/CLS2-466-Set-feature-flag-for-task-reminder

## Description of change

Remove feature flag for task reminders and only use feature flag for Reminder field on form.

## Test instructions

Only users with the `adviser-tasks` feature flag set will be able to see the reminder fields. 
## Screenshots

### Without `adviser-tasks` feature flag

![image](https://github.com/uktrade/data-hub-frontend/assets/699259/516bce17-c775-490b-92bc-088670112c2c)

### With `adviser-tasks` feature flag

![image](https://github.com/uktrade/data-hub-frontend/assets/699259/c467c4b4-476d-4e3a-b82e-bc08efa85bb9)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
